web/download_manager.js

```diff
@@ -64,7 +64,12 @@ class DownloadManager {
     download(blobUrl, filename);
   }
 
-  download(blob, url, filename) {
+  /**
+   * @param isFallback {boolean} Used to signal that the download was triggered
+   *   from the fallback bar. The version of PDF.js integrated with Firefox uses
+   *   this to to determine whether to show the "save as" or "open with" dialog.
+   */
+  download(blob, url, filename, isFallback) {
```

Comment:
Please change this to the following, to clearly mark the parameter as optional:
```suggestion
  download(blob, url, filename, isFallback = false) {
```